src/Exception/NotAllowedTypeException.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Opstalent\CrudBundle\Exception;
+
+/**
+ * Class NotAllowedTypeException
+ * @author Szymon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle
+ */
+class NotAllowedTypeException extends \Exception
```

Comment:
why not `TypeNotAllowedException`?

Reply:
also it shouldn't extend `\Exception` it should extend `LogicException` or `RuntimeException`